run.sh

```diff
@@ -0,0 +1,3 @@
+npm install
+npm install es6-promise
```

Comment:
IMO it would be better to add this to [`package.json`](https://github.com/Mashape/mockbin/blob/master/package.json#L70)

Reply:
I thought of that but this would affect the application itself. I couldn't see any issue regarding using installable application (without docker) so thought it could be an issue with docker setup (like missing some libs in setup).

Reply:
Adding `es6-promise` resolved #62 because it is a dependency of this app from what I can tell. An `npm install mockbin` on a blank system yields the same error as the Docker build